examples/adder/Cargo.toml

```diff
@@ -10,14 +10,15 @@ crate-type = ["cdylib"]
 [dependencies]
 near-sdk = { path = "../../near-sdk", features = ["unstable"] }
 serde = { version = "1", features = ["derive"] }
+borsh = { version = "1.0.0-alpha.3", features = ["derive"] }
```

Comment:
Is there a way to avoid direct dependency on borsh (and serde), and keep using the re-exported version from near-sdk-rs?

Reply:
yes, i'll address it in a new pr into `borsh` as discussed out of band, and post link to it in this thread.

Reply:
Some extra-redundant cross-referencing: [pr](https://github.com/near/borsh-rs/pull/210)